<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServeNextPatient</class>
 <widget class="QDialog" name="ServeNextPatient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1398</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #111111;</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>571</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #00d2ff;</string>
   </property>
   <property name="text">
    <string>Clinic Managemnet System - Serve Next Patient </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #1c1c1c;  
border: 1px solid #333333;  
border-radius: 6px;
color: #ffffff;            
padding: 10px 15px;
font-size: 15px;
font-weight: bold;
transition: 0.45s ease;
}

QPushButton:hover {
background-color: #111111; 
border: 1px solid #00d2ff; 
}
</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>180</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
font-size: 28px</string>
   </property>
   <property name="text">
    <string>Patient Info</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>540</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent</string>
   </property>
   <property name="text">
    <string>Token Number</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>440</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent</string>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>290</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent</string>
   </property>
   <property name="text">
    <string>Patient Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="patient_age">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>390</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #1c1c1c;  
    border: 1px solid #333333;   
    border-radius: 4px;
    color: #ffffff;
    padding: 5px;
}
QLineEdit:focus {
    border: 1px solid #00d2ff;
    background-color: #222222;
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>390</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent</string>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>270</y>
     <width>611</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #1c1c1c;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>490</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent</string>
   </property>
   <property name="text">
    <string>Disease</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="patient_token_number">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>540</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #1c1c1c;  
    border: 1px solid #333333;   
    border-radius: 4px;
    color: #ffffff;
    padding: 5px;
}
QLineEdit:focus {
    border: 1px solid #00d2ff;
    background-color: #222222;
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="patient_gender">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>440</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #1c1c1c;  
    border: 1px solid #333333;   
    border-radius: 4px;
    color: #ffffff;
    padding: 5px;
}
QLineEdit:focus {
    border: 1px solid #00d2ff;
    background-color: #222222;
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="patient_name">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>290</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #1c1c1c;  
    border: 1px solid #333333;   
    border-radius: 4px;
    color: #ffffff;
    padding: 5px;
}
QLineEdit:focus {
    border: 1px solid #00d2ff;
    background-color: #222222;
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="patient_disease">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>490</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #1c1c1c;  
    border: 1px solid #333333;   
    border-radius: 4px;
    color: #ffffff;
    padding: 5px;
}
QLineEdit:focus {
    border: 1px solid #00d2ff;
    background-color: #222222;
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>340</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent</string>
   </property>
   <property name="text">
    <string>Doctor's Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="doctors_name">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>340</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #1c1c1c;  
    border: 1px solid #333333;   
    border-radius: 4px;
    color: #ffffff;
    padding: 5px;
}
QLineEdit:focus {
    border: 1px solid #00d2ff;
    background-color: #222222;
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>label_2</zorder>
  <zorder>back_btn</zorder>
  <zorder>label_8</zorder>
  <zorder>label_7</zorder>
  <zorder>label_5</zorder>
  <zorder>label</zorder>
  <zorder>patient_age</zorder>
  <zorder>label_4</zorder>
  <zorder>label_6</zorder>
  <zorder>patient_token_number</zorder>
  <zorder>patient_gender</zorder>
  <zorder>patient_name</zorder>
  <zorder>patient_disease</zorder>
  <zorder>label_3</zorder>
  <zorder>doctors_name</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
